pysb/examples/bngwiki_egfr_simple.py

```diff
@@ -39,7 +39,7 @@
 
 
 Monomer('EGF', ['R'])
-Monomer('EGFR', ['L','CR1','Y1068'], {'Y1068':('U','P')})
+Monomer('EGFR', ['L','CR1','Y1068'], {'Y1068': ['U', 'P']})
```

Comment:
Maybe Monomer should just normalize `sites` and `site_states.values()` to `list`, but that can be done outside of this PR.

Reply:
Agree, we should normalize in `core.py`.